### ISSUE TITLE
After updating to 10.2.0 suddenly all email alerts are light themes

### ISSUE THREAD

**Issue by electrofloat** [grafana/grafana#77175]
### What happened?

I've just updated to Grafana v10.2.0 (895fbafb7a) and all the email alerts I'm getting suddenly are light themed.

It was dark themed before, and I cannot see a single setting to set it back to dark.

### What did you expect to happen?

To stay dark themed.

### Did this work before?

Yes, it was dark.

### How do we reproduce it?

1. Upgrade to Grafana v10.2.0 (895fbafb7a), and see that alert mails are light themed.
2.
3.

### Is the bug inside a dashboard panel?

No

### Environment (with versions)?

Grafana:Grafana v10.2.0 (895fbafb7a)
OS: Linux

### Grafana platform?

Docker

**Comment by kenntun1** [comment]
Is it possible to add options for setup email as dark or white?
As some alert may need to using dark and some need white.

**Comment by electrofloat** [comment]
https://github.com/grafana/grafana/pull/72398

**Comment by kenntun1** [comment]
> Hi,
> 
> Thanks for opening this issue.
> 
> Were both of you using the default email template or did any customization?

I am usng default email template without any customization
WIll be great if email theme can be select base on contact points or alert rules.

**Comment by gillesdemey** [comment]
Hi – I hear you on the dark theme—it was pretty slick. We only have the bandwidth to maintain a single email template and theme right now and we've decided to go with a light one, several email clients support either full or partial color inversions but there's not a single solution that works for everyone.

Right now, we're all hands on deck with some other stuff that's keeping us pretty busy. So adding new themes isn't in the cards at the moment. We've made a note of your suggestion, and it's on our radar for when we plan out future updates.
